src/util.ts

```diff
@@ -12,4 +14,24 @@ const aqiCutoffs = {
   veryUnhealthy: 300, // Very unhealthy for all individuals (201-300)
 };
 
+/**
+ * Interface for a Selected Sensor. This is used in the Home page to keep track
+ * of the values for the sensor that is currently selected on the map.
+ * - purpleAirId: The PurpleAir ID of the sensor
+ * - sensorDocId: the ID of the document for this sensor in the "sensors" collection
+ * - name: The name of this sensor on PurpleAir
+ * - aqi: The current AQI value of this sensor
+ * - isValid: boolean, is the sensor currently displaying an AQI
+ * - lastValidAqiTime: the last time this sensor reported a valid AQI
+ */
+interface SelectedSensor {
+  purpleAirId: number;
+  sensorDocId: string;
+  name: string;
+  aqi: string;
```

Comment:
Why is AQI a string?

Reply:
it's always been a string, so I just maintained it because that's how it's being used in other parts of the code. I can change it to a number and update the rest of the codebase to reflect that if you think that'd be better. Thoughts?

Reply:
Where do you first set the `aqi`? If it's take from Firestore and then immediately converted to a 2-digit string, then I think leaving it as a string makes sense